bpfman/src/bin/cli/args.rs

```diff
@@ -307,9 +312,241 @@ pub(crate) struct GetArgs {
 
 #[derive(Subcommand, Debug)]
 #[command(disable_version_flag = true)]
+#[allow(clippy::large_enum_variant)]
 pub(crate) enum ImageSubCommand {
     /// Pull an eBPF bytecode image from a remote registry.
     Pull(PullBytecodeArgs),
+    /// Build an eBPF bytecode image from local bytecode objects, and push to a registry.
+    Build(BuildBytecodeArgs),
+    /// Generate the OCI image labels for a given bytecode file.
```

Comment:
```suggestion
    /// Generate the OCI image labels for a given bytecode file.
    ///
    /// To use, the bytecode object file must be included, which can be passed in several ways.
    /// Use either:
    /// * --bytecode: for a single image of host architecture.
    /// * --cilium-ebpf-project: to pass a path which contains multiple object files for
    ///   different architectures.
    /// * --bc-386-el .. --bc-s390x-eb: to add one or more architecture object files.
    ///
    /// Examples:
    ///   bpfman image generate-build-args --bc-amd64-el ./examples/go-xdp-counter/bpf_x86_bpfel.o
```
How will the output data be used?

Reply:
We're currently using it in actions so that we can still build/push with the default docker actions rather than using bpfman for it all 

This means that we can use the existing tag management automation we already have in place saving us a bunch of headaches.